modules/core/arrow-typeclasses/src/main/kotlin/arrow/typeclasses/MonadError.kt

```diff
@@ -131,4 +131,14 @@ interface MonadThrow<F> : MonadError<F, Throwable> {
     wrapReturn.startCoroutine(continuation, continuation)
     return continuation.returnedMonad()
   }
+
+  override fun <B> binding(c: suspend MonadContinuation<F, *>.() -> B): Kind<F, B> =
```

Comment:
No, but this does not need to be annotated with `@Deprecated` because this is an override and the most super declaration already is annotated as `@Deprecated`

Reply:
Would I get the deprecation warning on the IDE if I reference it directly as Try.monadError().binding {} ?

Reply:
![screen shot 2019-01-24 at 3 09 15 pm](https://user-images.githubusercontent.com/456796/51683398-11072500-1fea-11e9-91e6-ee2a448a6d6c.png)
Yep